### ISSUE TITLE
 [BUG] WCAG AAA violations color contrast errors reported for black text on white background

### ISSUE THREAD

**Issue by CorinaMurg** [microsoft/playwright#28341]
### System info
- Playwright Version: v1.40.0
- Operating System: Windows 10
- Browser: All
- Other info:

### Source code

- [x] I provided exact source code that allows reproducing the issue locally.

**Config file**

```js
// playwright.config.ts
import { defineConfig, devices } from'@playwright/test';
import process from 'process';

export default defineConfig({
  testDir: './tests',
  fullyParallel: true,
  forbidOnly: !!process.env.CI,
  retries: process.env.CI ? 2 : 0,
  workers: process.env.CI ? 1 : undefined,
  reporter: 'html',
  use: {
    trace: 'on-first-retry',
  },
  projects: [
    {
      name: 'chromium',
      use: { ...devices['Desktop Chrome'] },
    },
    {
      name: 'firefox',
      use: { ...devices['Desktop Firefox'] },
    },
    {
      name: 'webkit',
      use: { ...devices['Desktop Safari'] },
    },
  ],
});
```

**Test file (self-contained)**

```js
import { test, expect } from '@playwright/test';
import AxeBuilder from '@axe-core/playwright';

test('should not have any automatically detectable WCAG A or AA violations', async ({ page }) => {
    await page.goto('https://corinamurg.netlify.app/');
  
    const accessibilityScanResults = await new AxeBuilder({ page })
        .withTags(['wcag2a', 'wcag2aa', 'wcag21a', 'wcag21aa'])
        .analyze();
  
    expect(accessibilityScanResults.violations).toEqual([]);
});

```
**CSS snippet:**
```js
.sample-project--description {
    background-color: #fff;
    color: #000;
    font-size: 1.2rem;
    font-weight: 400;
    margin-top: 0;
}
```

**Steps**
- Run the test in UI mode

**Expected**

Expected no violations for this element, given the high contrast between text and background: text color is black (#000) and background is white (#fff).

**Actual**

```
 "failureSummary": "Fix any of the following:
+   Element has insufficient color contrast of 2.12 (foreground color: #b2b2b2, background color: #ffffff, font size: 14.4pt (19.2px), font weight: normal). Expected contrast ratio of 4.5:1",
+         "html": "<p class=\"sample-project--description\">React, Typescript</p>",
+         "impact": "serious",
+         "none": Array [],
+         "target": Array [
+           ".my-portfolio-site > .sample-project > .sample-project--description",
+         ],

```

**Comment by mxschmitt** [comment]
This sounds like a bug which should be reported at [@axe-core/playwright](https://www.npmjs.com/package/@axe-core/playwright) or [axe-core](https://github.com/dequelabs/axe-core) since Playwright does not do anything related to axe.
